webapp/pages/admin/donations.vue

```diff
@@ -55,14 +57,25 @@ export default {
   methods: {
     submit() {
       const { showDonations } = this
-      const { goal, progress } = this.formData
+      let { goal, progress } = this.formData
+      goal = typeof goal === 'string' && goal.length > 0 ? goal : '15000'
```

Comment:
Why a default of `15000`? That seems oddly specific

Reply:
What would you do here?
We have to have a default value here at least for showing the donations progress bar on the main page I think.